lib/statsd.js

```diff
@@ -311,12 +311,9 @@ Client.prototype.send = function (message, tags, callback) {
     throw this.dnsError;
   }
 
-  var mergedTags = [];
+  var mergedTags = this.globalTags;
   if(tags && Array.isArray(tags)){
-    mergedTags = mergedTags.concat(tags);
-  }
-  if(this.globalTags && Array.isArray(this.globalTags)){
-    mergedTags = mergedTags.concat(this.globalTags);
+    mergedTags = overrideTags(mergedTags, tags);
```

Comment:
I believe this changes things not just for child clients but also for adding tags over global tags.  I'm fine with this, it makes sense to me to do this.  I will though bump it to a major version for this though as it is a change in functionality.